plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2575,7 +2575,7 @@ public void testAllAvailableTypes()
         assertUpdate("DROP TABLE test_all_types");
     }
 
-    @Test
+    @Test(timeOut = 5000)
```

Comment:
nit : I would keep timeout something between 20-30 sec since CI workers are small nodes and we do encounter GC pauses

Reply:
Sure, bumped it to 25s